description: Disables the Google URL Tracker, which is a service that seeks the address to Google for background communication

--- a/components/google/core/browser/google_url_tracker.cc
+++ b/components/google/core/browser/google_url_tracker.cc
@@ -135,6 +135,9 @@ void GoogleURLTracker::StartFetchIfDesirable() {
   //
   // See comments in header on the class, on RequestServerCheck(), and on the
   // various members here for more detail on exactly what the conditions are.
+  already_fetched_ = true;
+  need_to_fetch_ = false;
+  return;
   if (in_startup_sleep_ || already_fetched_ || !need_to_fetch_)
     return;
 

